extensions/wikia/AdEngine/js/lookup/prebid/prebidHelper.js

```diff
@@ -0,0 +1,34 @@
+define('ext.wikia.adEngine.lookup.prebid.prebidHelper', function() {
+	var adUnits = [];
+
+	function getAdUnits(adapter, slots) {
+		var adaptersAdUnits = [];
+
+		Object.keys(slots).forEach(function(slotName) {
+			adaptersAdUnits.push(adapter.prepareAdUnit(slotName, slots[slotName]));
```

Comment:
I added the `getSlots` method because I wanted to remove this piece of code from the adapters:
```
var skinSlots = slots[skin],
    adUnits = [];

Object.keys(slots[skin]).forEach(function(slotName) {
	adUnits.push(prepareAdUnit(slotName, skinSlots[slotName]));
})
```

this method was in AppNexus adapter and I needed to add the same one to IndexExchange. Instead I decided to add getSlots and do execute this shared logic in helper to avoid code duplication and to make it easier for next adapters (less copy and paste between them)